monkey/infection_monkey/system_info/collectors/scoutsuite_collector/scoutsuite_collector.py

```diff
@@ -26,5 +27,5 @@ def run_scoutsuite(cloud_type: str):
                               aws_session_token=WormConfiguration.aws_session_token)
 
 
-def send_results(results):
+def send_results(results: AWSProvider):
```

Comment:
Something doesn't make sense if you have a var called `results` of type `AWSProvider`

Reply:
You're right